src/ui/components/DataTable/CustomizeColumns.tsx

```diff
@@ -1,134 +1,151 @@
+	};
+
+	const restore = async () => {
+		await toWorker("main", "updateColumns", {
+			columns: config.fallback,
+			key: config.tableName,
+		});
+		onHide();
+		onSave();
+	};
+
+	const save = async () => {
+		const enabledColumns: string[] = columns
+				.filter(c => !c.hidden)
+				.map(c => c.key),
+			currentColumns: string[] = config.columns.map(c => c.key);
+
+		// Find columns we need to remove, and columns we need to add
+		const removeColumns: string[] = difference(currentColumns, enabledColumns),
+			addColumns: string[] = difference(enabledColumns, currentColumns);
+
+		// Apply removals and adding to currentColumns while trying to preserve the order of currentColumns
+		const nextColumns: string[] = currentColumns.filter(
+			c => !removeColumns.includes(c),
+		);
+		nextColumns.push(...addColumns);
+
+		await toWorker("main", "updateColumns", {
+			columns: nextColumns,
+			key: config.tableName,
+		});
+		onHide();
+		onSave();
+	};
+
+	const colsGrouped: { [key: string]: ColConfig[] } = groupBy(
+		columns,
+		c => c.cat,
+	);
 
 	return (
-		<Modal animation={false} centered show={show} onHide={onHide}>
-			<Modal.Header closeButton>Customize Columns</Modal.Header>
+		<Modal
+			animation={false}
+			size={"lg"}
+			show={show}
+			onEscapeKeyDown={exit}
+			centered
+		>
+			<Modal.Header onHide={exit} closeButton>
+				Customize Columns
+			</Modal.Header>
 			<Modal.Body>
-				<p>
-					Click and drag to reorder columns, or use the checkboxes to show/hide
-					columns.
-				</p>
-				{hasSuperCols ? (
-					<p className="text-danger">
-						This table has two header rows. That means you can enable/disable
-						columns, but not reorder them.
-					</p>
-				) : null}
-				<Container
-					helperClass="sort-inside-modal"
-					isDragged={isDragged}
-					onSortStart={() => {
-						setIsDragged(true);
-					}}
-					onSortEnd={args => {
-						setIsDragged(false);
-						if (!hasSuperCols) {
-							onSortEnd(args);
-						}
-					}}
-				>
-					{colOrder.map(({ colIndex, hidden }, i) => {
-						const col = cols[colIndex];
-						return (
-							<Item
-								key={colIndex}
-								index={i}
-								onToggleHidden={onToggleHidden(i)}
-								hidden={hidden}
-								col={col}
-							/>
-						);
-					})}
-				</Container>
+				<ul className="list-group list-group-flush">
+					{Object.entries(colsGrouped).map(([group, cols]) => (
+						<li key={group} className="list-group-item">
+							<strong>{group}</strong>
+							<div className="row">
+								{cols.map((col, i) => (
+									<div key={i} className="col-lg-4">
+										<div className="form-check d-flex">
+											<input
+												id={`show-column-${col.key}`}
+												className="form-check-input cursor-pointer"
+												type="checkbox"
+												checked={!col.hidden}
+												onChange={onChange(col.key)}
+											/>
+											<label
+												className="form-check-label cursor-pointer user-select-none flex-grow-1"
```

Comment:
Took out `cursor-pointer` and `user-select-none`.

I intentionally added flex-grow-1 because if you're click the center of the text to toggle the check boxes, its a little weird to me that click in these 2 locations would only work on one of these checks (by default the label is only as long as the text).
![image](https://user-images.githubusercontent.com/45333542/150994015-12c392e0-21c3-4d66-8aa8-b4e52b5f42e6.png)

@dumbmatter It could be changed to like `min-width:100px;` but i can remove it if this doesn't really matter

Reply:
also curor-pointer was on the label to indicate clicking it did something

Reply:
`min-width:100px;` sounds fine. Labels generally focus/toggle the associated form field but don't have a pointer. Not sure why, but it's normal.

Reply:
Removed the cursor pointer stuff and added the min width but this is mostly solved by always showing the descriptions https://github.com/zengm-games/zengm/pull/406#discussion_r793755399